pkg/cmd/login.go

```diff
@@ -51,6 +51,7 @@ func NewLoginCmd(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVar(&tenantCfg.OIDC.ClientSecret, "oidc.client-secret", "", "The OIDC client secret, see https://tools.ietf.org/html/rfc6749#section-2.3.")
 	cmd.Flags().StringVar(&tenantCfg.OIDC.ClientID, "oidc.client-id", "", "The OIDC client ID, see https://tools.ietf.org/html/rfc6749#section-2.3.")
 	cmd.Flags().StringVar(&tenantCfg.OIDC.Audience, "oidc.audience", "", "The audience for whom the access token is intended, see https://openid.net/specs/openid-connect-core-1_0.html#IDToken.")
+	cmd.Flags().StringSliceVar(&tenantCfg.OIDC.Scopes, "oidc.scopes", []string{"offline_access"}, "Optional scopes, see https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest")
```

Comment:
Thanks a lot for the update @RaphaelBut. I think we could have a different approach though. It seems like not only `openid`, but also `offline_access` scopes are required, correct? 

We might also have to be careful about duplication of scopes, e.g. what if the user also provides the `openid` scope? We will end up appending another string with `openid` to the list of scopes. Could this break something? I think we should only append it when it's missing.

Reply:
Hi @douglascamata, thanks for the reviews btw

I am not sure if or in which use cases offline_access is required/desired but for service accounts it should explicitly not be required (or so I am told) and is the reason why I want this flag to override. 

My understanding is without offline_access the application is not able to request a new token on its own and the user has to authenticate again if the token expires. If obsctl is used by a service account during a pipeline this wont be needed and is the reason why offline_access is not present on new service accounts. 

Regarding the second point, I can update this to be cleaner and prevent duplicated scopes in the request but would like to keep the offline_access as default on the flag to not introduce any change. 

(update: just tested it locally and it does not seem to change anything if we specify no scope or duplicates
maybe the golang.org/x/oauth2 package sorts it out under the hood :thinking: )

Reply:
Hey @RaphaelBut! I think what you say makes sense. I like the simplified option for the offline access scope. Thanks a lot for the research.